src/azure-cli/azure/cli/command_modules/synapse/commands.py

```diff
@@ -478,5 +482,13 @@ def get_custom_sdk(custom_module, client_factory):
         g.custom_command('export', 'export_notebook')
         g.custom_command('delete', 'delete_notebook', confirmation=True, supports_no_wait=True)
 
+    # Data Plane Commands --Managed private endpoints operations
+    with self.command_group('synapse managed-private-endpoints', synapse_managed_private_endpoints_sdk,
+                            custom_command_type=get_custom_sdk('managedprivateendpoints', None)) as g:
+        g.custom_show_command('show', 'get_Managed_private_endpoints')
+        g.custom_command('create', 'create_Managed_private_endpoints', supports_no_wait=True)
+        g.custom_command('list', 'list_Managed_private_endpoints')
+        g.custom_command('delete', 'delete_Managed_private_endpoints', confirmation=True, supports_no_wait=True)
```

Comment:
Both `create` and `delete` support no wait but there seems no `wait` command

Reply:
api doesn't suport wait command, so delete supports_no_wait=True.